nano/node/network.cpp

```diff
@@ -476,6 +477,17 @@ class network_message_visitor : public nano::message_visitor
 		{
 			node.logger.try_log (boost::str (boost::format ("Received confirm_ack message from %1% for %2% timestamp %3%") % channel->to_string () % message_a.vote->hashes_string () % std::to_string (message_a.vote->timestamp ())));
 		}
+
+		std::chrono::system_clock::time_point timestamp{ std::chrono::milliseconds{ message_a.vote->timestamp () } };
```

Comment:
An std::chrono::duration inmy ststem uses signed long to store the time duration.
Not all 64-bit values would translate as expected. I think it would simpler to do the calculations using 64 bit arithemtic.